### DISCUSSION TITLE
Just Completed Blockchain Basics Course

### DISCUSSION THREAD

**Discussion by Harshvardhantomar** [Cyfrin/foundry-full-course-cu#1583] [General]
Hey Everyone, I Just took the first step towards my blockchain journey. Completed the 3-hours long, Blockchain Basics course on Cyfrin Updraft. Excited to learn and also work with everybody.
(P.S. I am new to this field so I might have silly questions. Please guide me and support me. Looking forward to connecting with the community.)

**Comment by EngrPips** [comment]
I wish you the best on your journey of knowledge !
